### DISCUSSION TITLE
自建服务器支持 https 吗，证书应该如何配置

### DISCUSSION THREAD

**Discussion by c0nstantien** [chanify/chanify#74] [Q&A]
请问如果自建节点服务器，支持 endpoint 为 https 协议吗，支持的话证书应该如何配置。

**Comment by wizjin** [comment]
你好，自建节点服务器本身不支持直接使用 https 协议，但是可以在前面配置例如 nginx 之类的 web 代理来支持 https。

endpoint 本身可以使用 https 开头的。

谢谢

**Reply by c0nstantien** [reply]
OK，感谢！
